src/ParameterPostProcessor.php

```diff
@@ -49,7 +45,7 @@ public function __invoke(array $config) : array
         return $config;
     }
 
-    private function convertValues(array $values, string $prefix = '') : array
+    private function resolveNestedParameters(array $values, string $prefix = '') : array
```

Comment:
As we do not have any changes in this method do we need to change the name?
* Was the name incorrect before?
* Is the new name better and more clear what the function is actually doing?